db/schema.rb

```diff
@@ -74,8 +74,8 @@
     t.string "reference", null: false
     t.string "move_type"
     t.string "additional_information"
-    t.integer "nomis_event_id"
     t.datetime "time_due"
+    t.integer "nomis_event_id"
```

Comment:
Probably you want to revert just this line, as it seems to be wrong if you do a `rails db:drop db:create db:migrate` command